src/BuildingBlocks/StackSdks/Masa.BuildingBlocks.StackSdk.Tsc/Model/Log/PaginationDto.cs

```diff
@@ -0,0 +1,23 @@
+// Copyright (c) MASA Stack All rights reserved.
+// Licensed under the MIT License. See LICENSE.txt in the project root for license information.
+
+namespace Masa.BuildingBlocks.StackSdks.Tsc.Contracts.Model;
+
+public sealed class PaginationDto<TModel>
```

Comment:
Should inherit PaginatedListBase<TEntity>

Reply:
Is not used, has removed.